Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/Surfaces/EnhancedSurface.azsli

```diff
@@ -13,6 +13,21 @@
 #include <Atom/Features/PBR/Surfaces/ClearCoatSurfaceData.azsli>
 #include <Atom/Features/PBR/Surfaces/TransmissionSurfaceData.azsli>
 
+// This data varies across different surfaces, but is uniform within a surface
+class SurfaceSettings
```

Comment:
Not entirely clear from the name/comment why SurfaceSettings is separate from Surface. Doesn't Surface also vary across surfaces but stay uniform within a surface? Either way this isn't anything to hold up the PR for, but a more explanatory comment could help.

Reply:
Basically, for a given material, the data in `SurfaceSettings` is constant but the data in `Surface` changes across fragments. Does that help? I can add more explanation if needed